lib/constants/enums.dart

```diff
@@ -16,3 +16,16 @@ enum MessageDirection {
   received,
   unset,
 }
+
+enum GroupIdent {
```

Comment:
There are more group idents in addition to these, but these are the "Core groups" for mentors/mentees. 

IMO this is fine, just wanted to add a note for context